orchestrator/db/models.py

```diff
@@ -408,10 +404,6 @@ class WorkflowTable(BaseModel):
     )
     processes = relationship("ProcessTable", lazy="select", cascade="all, delete-orphan", back_populates="workflow")
 
-    @staticmethod
-    def find_by_workflow_name(name: str) -> WorkflowTable:
-        return WorkflowTable.query.filter(WorkflowTable.name == name).scalar()
```

Comment:
Are we sure these removed methods aren't used by others?

Reply:
No. This would be a breaking change.
Unfortunately, we can't rewrite this to use db.session. That would defeat the purpose of the refactor and create a cyclic dependency between modules

But there's an easy migration: Just use `orchestrator.services.workflows.get_workflow_by_name`

Reply:
Ok, makes sense. We'll have to remember to put this in the release notes then.